_posts/2024-05-14-switch-to-bam.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: What does it take to switch over to bam from fastq?
+tags:
+  - bam
+  - fastq 
+contributors:
+  - lskatz
+  - SeanSierra-Patev
+reviewers:
+  - pmenzel
+---
+
+It has been over 14 years since the formalization of the fastq format {% cite Cock2009 %}.
```

Comment:
```suggestion
It has been over 14 years since the formalization of the fastq format ({% cite Cock2009 %}).
```

Reply:
I'll update this manually to help avoid a git collision